bound/kt/src/main/kotlin/web5/sdk/crypto/Ed25519Generator.kt

```diff
@@ -14,8 +16,14 @@ class Ed25519Generator {
          * @return Jwk the JWK with private key material included.
          */
         fun generate(): Jwk {
```

Comment:
chad says:
You can simplify the generate() function by removing the redundant catch block for the generic Exception, as it will automatically be propagated if not caught.

Reply:
also since there are a lot of these it could be cleaner to do it this way with return try:
```
    return try {
        val rustCoreJwkData = ed25519GeneratorGenerate()
        Jwk.fromRustCoreJwkData(rustCoreJwkData)
    } catch (e: RustCoreException) {
        throw Web5Exception.fromRustCore(e)
    }
```